### ISSUE TITLE
"Link Channels" does not include choice of progression

### ISSUE THREAD

**Issue by musikBear** [LMMS/lmms#4936]
RC8 (most likely all -Tested in rc6 & 8)
Recreate:
* Insert any FX with split channels (f.i. GLame LP Filter)
* Make sure *link-channels* is On (default) 
* in *Controls* set left-channel *CutOff* to *Linear* via context-menu (default is Log)
* turn dial
The right channel Dial will still be logarithmic and appear to be out of sync with the left one, witch it is not, it's just not Linear as the left one.

**Comment by JohannesLorenz** [comment]
I guess this should be milestoned stable-1.2 . Any vetos?

**Comment by DomClark** [comment]
I don't think it should be milestoned: it's only a UI/UX issue, doesn't affect the stability of the program nor the quality of the audio, and isn't a regression from 1.1.3 (same behaviour there). I have no issues with it being fixed in stable-1.2, but I don't think it should block the release.
